docs/content/Workspace/Inspecting-Pre-aggregations.mdx

```diff
@@ -0,0 +1,83 @@
+---
+title: Pre-aggregation inspection
+permalink: /cloud/inspecting-pre-aggregations
+category: Workspace
+menuOrder: 5
+---
+
+The Pre-Aggregations page in Cube Cloud allows you to inspect all
+pre-aggregations in your deployment at a glance. You can see which
+pre-aggregations are accelerating queries, which are not, and which are not
+being refreshed, along with the last 24 hours of build history.
+
+<Screenshot src="https://ucarecdn.com/eb10de99-4ff6-4a9f-abd5-736971497583/" />
+
+## Exploring pre-aggregations
+
+You can switch between [pre-aggregations](#pre-aggregations) and
+[build history](#build-history) by switching between the&nbsp;
+&ZeroWidthSpace;<Btn>Pre-Aggregations</Btn> and <Btn>Build History</Btn> tabs.
+
+## Pre-aggregations
+
+The <Btn>Pre-Aggregations</Btn> tab shows all configured pre-aggregations for
+the deployment. To see more information about a specific pre-aggregation, click
+on it.
+
+### <--{"id" : "Pre-aggregations"}--> Partitions
+
+The <Btn>Partitions</Btn> tab shows all partitions of the pre-aggregation. In
+the example below, the pre-aggregation has a single partition:
+
+<Screenshot src="https://ucarecdn.com/d65d5910-3561-4101-9317-2c9f8366694e/" />
+
+### <--{"id" : "Pre-aggregations"}--> Definition
+
+The <Btn>Definition</Btn> tab shows how the pre-aggregation is defined in the
+data model:
+
+<Screenshot src="https://ucarecdn.com/4e8f203a-bd6b-4316-86ff-b742ccbc78a4/" />
+
+### <--{"id" : "Pre-aggregations"}--> Preview
+
+The <Btn>Preview</Btn> tab shows a sample of the data in the pre-aggregation:
+
+<Screenshot src="https://ucarecdn.com/7f5059bb-3bc9-492e-9c69-3b614f3feca5/" />
+
+### <--{"id" : "Pre-aggregations"}--> Used By
+
+The <Btn>Used By</Btn> tab shows recent queries that used this pre-aggregation
+for acceleration:
+
+<Screenshot src="https://ucarecdn.com/1dcef348-af33-434a-9ef7-7a4c5f30c26c/" />
+
+### <--{"id" : "Pre-aggregations"}--> Indexes
+
+The <Btn>Indexes</Btn> tab displays any configured indexes for this
+pre-aggregation:
+
+<Screenshot src="https://ucarecdn.com/f7e306e2-4245-46b4-8785-3e7fc1be518e/" />
+
+### <--{"id" : "Pre-aggregations"}--> Rebuilding
+
+If the pre-aggregation requires rebuilding (usually due to the underlying data
+changing in a significant way), click <Btn>Build All</Btn> to rebuild all
+partitions of the pre-aggregation:
+
+<Screenshot
+  highlight="inset(38% 1% 57% 91% round 10px)"
+  src="https://ucarecdn.com/d65d5910-3561-4101-9317-2c9f8366694e/"
+/>
```

Comment:
@hassankhan Wow, I see what you did here! Super cool! 😍

What if we, subjectively, simplify the configuration and, also subjectively, make the result more appealing by using a radial gradient (https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Images/Using_CSS_gradients#using_radial_gradients) for the highlight, providing just the coordinates of the center of the highlight, and its radius? I think it might make the screenshots more appealing but not sure.

Reply:
I tried playing around with it, but my CSS-fu is weak. Maybe we can ask @ewgenius to improve this at a later date?